data/Data Science Projet_data1_v2.R

```diff
@@ -0,0 +1,48 @@
+##read files
+temperature_nsw <- read.csv("C:/Users/howar/OneDrive/Documents/Data Science Project/temperature_nsw.csv", header=T);
+totaldemand_nsw <- read.csv("C:/Users/howar/OneDrive/Documents/Data Science Project/totaldemand_nsw.csv", header=T);
+forecastdemand_nsw <- read.csv("C:/Users/howar/OneDrive/Documents/Data Science Project/forecastdemand_nsw.csv", header=T);
+
+##load relevant libraries
+library(lubridate);
+library(dplyr);
+library(crayon);
+library(tidyr);
+
+##need to change to full outer join
+data <- totaldemand_nsw %>% full_join(temperature_nsw,by=c('DATETIME')) %>% select(-LOCATION,-REGIONID);
+
+##filter rows that do not have actual demand
+data <- data %>% filter(!is.na(TOTALDEMAND))
+
+##arrange by date in ascending
+data <- data %>% arrange(DATETIME);
+
+##fill in missing temperature (down fill)
+data <- data %>% fill(TEMPERATURE)
+
+##convert DATETIME to date
+data$DATETIME = ymd_hms(data$DATETIME);
```

Comment:
[no change required]: would be good to be consistent with the syntax below 
`data %>% mutate(DATETIME = ymd_hms(DATETIME)`